app/database/user_item_store_compute_all_user_items.go

```diff
@@ -120,15 +120,20 @@ func (s *UserItemStore) ComputeAllUserItems() (err error) {
 					LEFT JOIN items_items
 						ON items_items.idItemParent = users_items.idItem
 					SET
-						users_items.sLastActivityDate = IF(task_children_data.idUserItem IS NOT NULL AND items_items.ID IS NOT NULL,
+						users_items.sLastActivityDate = IF(task_children_data.idUserItem IS NOT NULL AND
```

Comment:
I don't get the need for this change. Can you tell me more about it?

Reply:
We modify fields depending on `children_data` only if `children_data` row exists. We didn't check that explicitly earlier and it worked somewhat correctly in MySQL 5.6, but not in MySQL 8.0